lib/service/servicecfg/config.go

```diff
@@ -308,6 +311,24 @@ func WithKubeMultiplexerIgnoreSelfConnectionsOption() KubeMultiplexerIgnoreSelfC
 	return KubeMultiplexerIgnoreSelfConnectionsOption{}
 }
 
+// AccessGraphConfig represents TAG server config
+type AccessGraphConfig struct {
+	// Enabled Access Graph reporting enabled
+	Enabled bool
+
+	// When enabled, the TAG server will be used to evaluate access requests
+	UseAuth bool
```

Comment:
This is a bit confusing tbh, so "Enabled" just turns on syncing of users/roles/resources to TAG, but "UseAuth" turns on actual RBAC checks? Would it make sense to rename these to something like `SyncEnabled bool` and `RBACEnabled bool` to make it more clear?

Reply:
My bad. "UseAuth" should be removed, as we're not shipping this functionality.